### PR TITLE
Restructuring configuration

### PR DESCRIPTION
**Pull request by adelq**
This is for ease of deployment on Dokku, and as a side effect has much simpler setup for development as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,20 +5,12 @@ Common Funding Application
...
-
-* Install [CoffeeScript](http://coffeescript.org) (`sudo npm install -g coffee-script`)
+* Create a virtualenv (`virtualenv env`) and activate it (`source env/bin/activate`). You will need to re-activate the virtualenv in every new terminal you use.
 
-* Compile the CoffeeScript files (`coffee -o app/static/js/ -c app/static/coffeescripts/` or `cake build` if you are in the app/static directory)
-
-* Create a configuration file (`cp sandbox_config.py_default sandbox_config.py`)
-
-* Fill in missing fields in sandbox_config.py
+* Install python dependencies (`sudo pip install -r requirements.txt`)
```
Comment (esqu1) [inline README.md]: sudo pip in a virtualenv?

Comment (adelq) [inline README.md]: Nice catch

**Comment by esqu1** [comment]
LGTM

**Comment by adelq** [comment]
Success! https://cfa.apps.pennlabs.org/

**Comment by adelq** [comment]
So, I'm not sure if these changes will work on the old server, and there's no easy way of finding out. So we need to make a choice about if and when we will transition CFA over to dokku.
1. If the answer is "soon", then we can just migrate over and do not need to check compatibility with the old server.
2. However, if the answer is "not soon" or "never", then this pull request needs to be kept on hold.

Decision should be made sooner rather than later because it contains breaking changes and will require a new release.

**Comment by adelq** [comment]
@esqu1 this is now deployed on penncfa.com, so the answer is "now" and we can get rid of the old configuration and the old server.

**Pull request merged**